### PR TITLE
Milestone 3: Leaderboard final touches

### PR DESCRIPTION
**Pull request by odhiambo-ed**
In this milestone, I did the following:

    - Added more style.
    - Did final touches.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -25,17 +25,20 @@
...
+                <div class="d-flex flex-row justify-content-between">
+                    <h2>Recent Score</h2>
+                    <button type="button" id="getter" class="btn btn-primary">Refresh</button>
+                    <!-- <div class="col">
                         <h2>Recent Score</h2>
                     </div>
                     <div class="col">
                         <button type="button" id="getter" class="btn btn-primary">Refresh</button>
-                    </div>
+                    </div> -->
```
Comment (Deepakdanger) [inline src/index.html]: - [ ] I would recommend removing the commented code to make  the codebase cleaner.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @white3d, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

- [ ] I would recommend to update the `live link` to make the README professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @white3d  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

the-simpsons-homer-simpson

 ## Required Changes ♻️

- [ ] Kindly update your live link.

- [ ] After cloning and installing dependencies I am having difficulties to run the project due to this error `Module not found: Error: Can't resolve '@popperjs/core' in '/Users/aimemalaika/Documents/Microverse/reviews/Learderboard-White3d/node_modules/bootstrap/dist/js'`, Kindly solve the issue or document in your PR description the way to solve the issue.
Screen Shot 2022-04-07 at 13 14 07



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Hi @white3d,
# Status: Approved ✔️ ✔️ ✔️ 
 
Gu2dpSp0qee4C710M5kVsWN425J3-vv2w52by6


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**